config/main.py

```diff
@@ -2045,7 +2045,12 @@ def add(ctx, interface_name, ip_addr, gw):
             ctx.fail("'interface_name' is None!")
 
     try:
-        ipaddress.ip_network(unicode(ip_addr), strict=False)
+        net = ipaddress.ip_network(unicode(ip_addr), strict=False)
+        if '/' not in ip_addr:
```

Comment:
I checked this code and found that `str(net)` will give correct ip address with the correct mask. So this could be changed to
`ip_addr = str(net)`

Reply:
Good point. Changed it!